apps/chat/src/components/Files/FileManagerModal.tsx

```diff
@@ -175,6 +177,19 @@ export const FileManagerModal = ({
       searchQuery,
     ),
   );
+
+  const sharedWithMeRootFolders = useAppSelector((state) =>
```

Comment:
Please, add `sharedWithMeRootFolders` and `sharedWithMeRootFiles` into this fix (I suggest to merge fix into this branch and adapt for `shared with me`)

Reply:
Thank you for the suggestion!
Branch merged. Made suggested changes.